package.json

```diff
@@ -7,7 +7,7 @@
   "types": "dist/src/wrapper.d.ts",
   "scripts": {
     "build": "rm -rf dist wrapper.tgz && tsc --build --force && tracer=\"$(npm pack)\" && mv \"$tracer\" wrapper.tgz",
-    "prepublishOnly": "npm run prettier:fix && npm run lint && npm run test:unit && npm run test:components",
+    "prepublishOnly": "npm run prettier:fix && npm run lint && npm run test:unit",
```

Comment:
Why deleting the component tests?

Reply:
because its part of the component test not the prepublishOnly stage